website/static/css/custom.css

```diff
@@ -4,28 +4,28 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
- 
+
 /* your custom css */
 
 @font-face {
    font-family: "Averta";
    font-style: normal;
    font-display: swap;
-   src: local("Averta-Regular"), url('/fonts/averta-regular.ttf') format('truetype');
+   src: local("Averta-Regular"), url('/safe/fonts/averta-regular.ttf') format('truetype');
```

Comment:
this breaks it for local development

Reply:
https://stackoverflow.com/questions/5815452/how-to-use-relative-absolute-paths-in-css-urls

Reply:
so maybe using `../fonts/avert-...` might be "better"

Reply:
It looks fine to me. I took it from https://github.com/gnosis/conditional-tokens-docs/commit/3e74890604697dee6c1d0b31c200fba06173df10

Do you perhaps still have your `DOCS_PATH` set to `/` instead of `safe` in the `.env` file?

Reply:
ahhh you run it locally also in the subdirectory ... in this case it is fine :)